website/docs/index.html.markdown

```diff
@@ -139,6 +139,19 @@ variables in order to keep credential information out of the configuration.
   that Terraform can be tricked into writing secrets to a server controlled
   by an intruder. May be set via the `VAULT_SKIP_VERIFY` environment variable.
 
+* `create_intermediate_child_token` - (Optional) Set this to `false` to disable
+  creation of an intermediate ephemeral Vault token for Terraform to
+  use. This is strongly discouraged in most cases and environments because it
+  can result in the provided Vault token being exposed by Terraform's output.
```

Comment:
This is only in the case when `TF_LOG` is set to `debug`. We should probably mention that here.

Reply:
Done.